//例外02　try～catch～finally文で例外処理を行う

package p12;

public class StartUp02_catch {

	public static void main(String[] args) {
		// TODO Auto-generated method stub

	}

}
